examples/example_get_remote_scratchpad_status.py

```diff
@@ -30,8 +30,9 @@ def print_node_list(nodes):
     print("\nList of nodes:")
 
     id = 0
-    for node_id, node in nodes:
+    for node_id in list(nodes):
```

Comment:
Wouldn't an iterator be more efficient here, i.e. omitting the `list()` constructor altogether?

Reply:
No, this is exactly the reason why I'm making this change. List() will implicitly generate a copy that I need.
In fact, if you iterate on an object (dic here) that can be modified by someone else, you may end up with a _RuntimeException: dictionary changed size during iteration_

Reply:
OK